rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1161,13 +1161,10 @@ public byte[] getActivePowpegRedeemScript(Object[] args) {
         }
     }
 
-    public boolean increaseLockingCap(Object[] args) throws BridgeIllegalArgumentException {
+    public boolean increaseLockingCap(Object[] args) throws VMException {
```

Comment:
Should we be using `LockingCapIllegalArgumentException` here?

Reply:
agree

Reply:
No, I avoided doing that since `LockingCapIllegalArgumentException` was created to be used in LockingCap objects. Please, take a look at this thread:
https://github.com/rsksmart/rskj/pull/2662#discussion_r1700054480

Reply:
you are right wilmer, i got confused. But I think it was supposed to throw a BridgeIllegalArgumentException, not a VM one. That's what I think, and overall what I read from the thread.

Reply:
`LockingCapIllegalArgumentException` extends `VMException`. It doesn't depend on the `BridgeIllegalArgumentException`. The idea is to decouple to the bridge objects.

Reply:
Done!